cscs-checks/microbenchmarks/gpu/gpu_burn/gpu_burn_test.py

```diff
@@ -8,11 +8,11 @@
 import reframe as rfm
 import reframe.utility.sanity as sn
 
-from library.microbenchmarks.gpu.gpu_burn import GpuBurnBase
+from library.microbenchmarks.gpu.gpu_burn import GPU_burn
```

Comment:
I would vote for all lowercase everywhere. It's easier to type in the command line.